### PR TITLE
[py]: move python code to `test_locators.py` 

### PR DESCRIPTION
**Pull request by navin772**
### **User description**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Description

Moved python code of locators to `test_locators.py` and 

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

___

### **PR Type**
Tests, Enhancement

___

### **Description**
- Added new Python test file `test_locators.py` with comprehensive test cases for all Selenium locator types
- Created HTML test page with form elements to support locator testing
- Added HTTP server fixture in conftest.py to serve test files
- Updated documentation across multiple languages to reference the new test implementations
- Improved code examples by linking to actual test implementations instead of inline code

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Tests</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>locators.html</strong><dd><code>Add HTML test page for locator examples</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

examples/python/tests/elements/locators.html

<li>Added new HTML test page with form elements for testing locators<br> <li> Contains various input fields, radio buttons, checkboxes and links<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/2102/files#diff-730a0ca74d035028c5e86953e142df2f3b8ffed8721579cd55c99ae35c10353f">+30/-0</a>&nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>test_locators.py</strong><dd><code>Add comprehensive locator tests in Python</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

examples/python/tests/elements/test_locators.py

<li>Added test cases for all locator types (class name, CSS, ID, etc.)<br> <li> Implemented WebDriver fixture for browser initialization<br> <li> Added assertions to verify element properties<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/2102/files#diff-37a84073184f55bf24b44b3a435c07704ca453e3d74fac87370df04efec76cb8">+52/-0</a>&nbsp; &nbsp; </td>

</tr>
</table></td></tr><tr><td><strong>Configuration changes</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>conftest.py</strong><dd><code>Add HTTP server fixture for serving test files</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

examples/python/tests/conftest.py

<li>Added HTTP server fixture to serve test HTML files<br> <li> Implemented server setup and teardown logic<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/2102/files#diff-046cf5671a09c47700a1903ecec9669341fe3c0602c91beb6bb529053e7af113">+20/-0</a>&nbsp; &nbsp; </td>

</tr>
</table></td></tr><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>locators.*.md</strong><dd><code>Update documentation to reference new Python test examples</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/elements/locators.*.md

<li>Updated Python code examples to reference new test file<br> <li> Replaced inline code with references to test implementation<br> <li> Applied changes across multiple language versions (en, ja, pt-br, <br>zh-cn)<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/2102/files#diff-efc549a3dbd8d87f9cc393a9c92280a95fa98461d530e04a80c8dd855f133d1a"></a></td>

</tr>
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

File: examples/python/tests/elements/locators.html
Comment (harsha509) [inline examples/python/tests/elements/locators.html:1]: We don't place HTML code here. Instead, you can use the static pages deployed on the official Selenium website
https://www.selenium.dev/selenium/web/

**Review by harsha509 — CHANGES_REQUESTED** [review]
HI Navin,

Thank you for the PR. Could you please look into comments provided.

Thanks,
Sri

File: examples/python/tests/elements/locators.html
Comment (navin772) [reply examples/python/tests/elements/locators.html:1]: Can't find a similar html page, so created a PR to add it in selenium - https://github.com/SeleniumHQ/selenium/pull/14905

Comment (navin772) [reply examples/python/tests/elements/locators.html:1]: Updated the PR to use - https://www.selenium.dev/selenium/web/locators_tests/locators.html which was added recently

**Review by yvsvarma — APPROVED** [review]
Thanks for these additions! They are well-structured and help clarify the documentation. Looks good to merge.

**Review by harsha509 — APPROVED** [review]

**Review by harsha509 — APPROVED** [review]
Thank you @navin772 !

**Pull request merged**